### PR TITLE
Enable metadata viewlet globally

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: sc/social/like/upgrades/v3046/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def reindex_catalog(setup_tool):
     for obj in get_valid_objects(results):
         catalog.catalog_object(obj, idxs=['object_provides'], update_metadata=False)
         n += 1
-        if n % 1000 == 0 and not test:
+        if n % 1000 == 0 and not test:  # noqa: S001
```
Comment (hvelarde) [inline sc/social/like/upgrades/v3046/__init__.py]: I think this is already fixed in a newer version of the flake8 plugin.

File: sc/social/like/browser/viewlets.py [hunk 2]
```diff
@@ -87,14 +87,8 @@ def render(self):
...
-        """Check if the viewlet should be shown in this context."""
-        template = self.helper.view_template_id()
-        # If using folder_full_view or all_content, we add metadata
-        # in order to proper display share buttons for
-        # contained content types
-        if template in ('all_content', 'folder_full_view',):
-            return True
-        return self.helper.enabled(self.view)
+        """Enable metadata viewlet globally"""
+        return True
```
Comment (hvelarde) [inline sc/social/like/browser/viewlets.py]: why are you doing this?

Comment (rodfersou) [reply sc/social/like/browser/viewlets.py]: these metadata should be used in other places where social like is not necessary.  Other plugins can use those metadatas.

Comment (rodfersou) [reply sc/social/like/browser/viewlets.py]: The tile likes we did in the new portal for example, we don't need social likes into the Cover type, but need the metadata to add these sharing options.

File: sc/social/like/browser/viewlets.py [hunk 3]
```diff
@@ -87,14 +87,8 @@ def render(self):
...
 
     def enabled(self):
-        """Check if the viewlet should be shown in this context."""
-        template = self.helper.view_template_id()
-        # If using folder_full_view or all_content, we add metadata
-        # in order to proper display share buttons for
-        # contained content types
-        if template in ('all_content', 'folder_full_view',):
-            return True
-        return self.helper.enabled(self.view)
```
Comment (hvelarde) [inline sc/social/like/browser/viewlets.py]: you can't remove this or you are going to introduce a bug; we need tests.

**Comment by rodfersou** [comment]
well maybe this is a bad idea..

I'll close this PR, if in the future we need this, let's open it again.

**Pull request closed without merging**